src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -382,31 +382,32 @@ private MLNodeAutoCompletionResponse GetMLNodeAutocompleteResults(string request
             MLNodeAutoCompletionResponse results = null;
             var authProvider = dynamoViewModel.Model.AuthenticationManager.AuthProvider;
 
-            if (authProvider is IDSDKManager manager)
+            if (authProvider is IOAuth2AuthProvider auth2AuthProvider)
             {
-                var uri = DynamoUtilities.PathHelper.getServiceBackendAddress(this, nodeAutocompleteMLEndpoint);
-                var client = new RestClient(uri);
-                var request = new RestRequest(Method.POST);
-
                 try
                 {
-                    manager.LoginRequest(ref request, client);
+                    RestRequest restRequest = new RestRequest();
```

Comment:
I think this deserves a quick comment noting why we're doing this, that it's a workaround, and that the types are different from the request and client types used below.

Reply:
Added a TODO comment.